src/lib/parser.ts

```diff
@@ -133,8 +133,8 @@ export function getLastSectionName(gptString: string): string {
 }
 
 export function parseFunctionCall(text: string): FunctionCall {
-  const functionCallRegex = /(\w+)\(([^)]*)\)/;
-  const argumentRegex = /([^,\s]+?)=({.*?}|'.*?'|".*?"|\[.*?\]|[^,]*)/g;
+  const functionCallRegex = /(\w+)\((.*)\)/;
+  const argumentRegex = /([^,\s]+?)=({.*?}|'.*?'|".*?([^\\])"|\[.*?\]|[^,]*)/g;
```

Comment:
Reckon these could use a comment now cos you need a PhD in regex to read them :D

Reply:
Haha fair point, I'll add one!